app/helpers/doc_helper.rb

```diff
@@ -1,7 +1,7 @@
 module DocHelper
 
   def man(name)
-    link_to name.gsub(/^git-?/, ''), doc_file_path(:file => name) 
+    link_to name.gsub(/^git-/, ''), doc_file_path(:file => name)
```

Comment:
I opened snh/git-scm.com#1 to fix this. If it looks good, merge it, and then I can merge this. Alternatively, I guess I could just merge this, and then do my fixup on top, and have a minor regression on master in between. That would probably be less complicated.